core/server/web/middleware/custom-redirects.js

```diff
@@ -27,6 +27,13 @@ _private.registerRoutes = function registerRoutes() {
              *   - you define /my-blog-post-1/ as from property
              *   - /my-blog-post-1 or /my-blog-post-1/ should work
              */
+
+            let options = '';
+            if (redirect.from.match(/\/.*\/i/)) {
```

Comment:
The comment above belongs to the condition below. Can you pls reorder? :)

A possible regex improvement could be: instead of `/\/.*\/i/`, use `/\/i$/`